src/Installers/Windows/AspNetCoreModule-Setup/ANCMV1/aspnetcoremodule.wxs

```diff
@@ -251,7 +252,7 @@
             <AppSearch Before="LaunchConditions" />
             <LaunchConditions After="FindRelatedProducts" />
             <MigrateFeatureStates />
-            <Custom Action="CheckForSharedConfiguration" After="InstallInitialize"></Custom>
+            <Custom Action="CheckForSharedConfiguration" After="InstallInitialize">DISABLESHAREDCONFIGCHECK=0</Custom>
```

Comment:
You'll need to consider conditioning this when the you are removing ANCM. When that happens from ARP, you cannot override the parameter unless the user is running an uninstall from the commandline in case removing it would case a problem

Reply:
Worth it to only do this on install?